<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FeAture Explorer (Research Used Only)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,1,1,1,2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFeatureExtraction">
         <property name="text">
          <string>Feature Extraction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonFeaturePreprocessing">
         <property name="text">
          <string>Feature Preprocessing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonModelExploration">
         <property name="text">
          <string>Model Exploration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonVisulization">
         <property name="text">
          <string>Visulization</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelIntroduction">
       <property name="text">
        <string>FeAture Explorer
 V.0.1

 Yang Song (songyangmri@gmail.com)
Jing Zhang (zhangjingmri@gmail.com)
Chengxiu Zhang (cxzhang@phy.ecnu.edu.cn)
Guang Yang (gyang@phy.ecnu.edu.cn)

 Shanghai Key Laboratory of Magnetic Resonance
East China Normal University
3663, Zhongshan North Road. Shanghai, China, 200062</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
